### DISCUSSION TITLE
<title>

### DISCUSSION THREAD

**Discussion by gaozhongquan** [community/community#44439] [Copilot Conversations]
### Select Topic Area

Bug

### Body

![image](https://user-images.githubusercontent.com/122274127/212601647-d6aaaefd-6b17-4a3d-ad2e-6a5b33923625.png)

**Comment by Akash1134** [comment]
Hi @gaozhongquan , It looks like you're using some kind of VPN software causing issues with the DNS server. Make sure your IDE is up-to-date and then disable/re-enable the GitHub Copilot extension and let us know how it goes. Thanks!

**Reply by gaozhongquan** [reply]
没有什么用。vscode使用很好没有这个问题
